steps/apply.ts

```diff
@@ -0,0 +1,27 @@
+import { Page } from '@playwright/test'
+import { DashboardPage } from '../pages/dashboardPage'
+import { ApplyPage, CRNPage, TaskListPage } from '../pages/apply'
+
+export const visitDashboard = async (page: Page): Promise<DashboardPage> => {
+  const dashboard = new DashboardPage(page)
+  await dashboard.goto()
+
+  return dashboard
+}
+
+export const enterCrn = async (dashboard: DashboardPage, page: Page, crn: string) => {
+  await dashboard.clickStartApplication()
+
+  const crnPage = new CRNPage(page)
+  await crnPage.enterCrn(crn)
+  await crnPage.clickSave()
+}
+
+export const completeFundingInformationTask = async (page: Page) => {
+  const taskListPage = new TaskListPage(page)
+  await taskListPage.clickTask('Funding information')
+
+  const fundingInformationPage = await ApplyPage.initialize(page, 'Funding information')
+  await fundingInformationPage.checkRadio('Personal money or savings')
+  await fundingInformationPage.clickSave()
```

Comment:
do you think it would be worth checking after this that we get back to the tasklist page, or that the save has worked?
I think this test would still pass even if the call to save errored. ([TIL about the handy debug methods in Playwright!](https://playwright.dev/docs/debug#run-a-test-from-a-specific-breakpoint))